source/about/staff.rst

```diff
@@ -13,11 +13,11 @@ operates efficiently.
 +-----------------------------------+------------------------------------+
 | Name                              | Role                               |
 +===================================+====================================+
-| blood                             | Implementation (Forge) Leader      |
+| Dualspiral                        | Sponge Leader                      |
```

Comment:
Why uppercase name?

Reply:
Actually, I just tweaked Gabizou to match the other two Leaders. The rest was already that way. We have no conventions at all on nick capitalisation, so it's more a matter of personal preference.

Reply:
@dualspiral Which do you prefer?

Reply:
All lowercase please.